modulefiles/ufs_aerosols_hera.intel

```diff
@@ -0,0 +1,22 @@
+#%Module
```

Comment:
How does this module get used? I would expect to see logic in `compile.sh` or `build.sh` if this module is indeed required.

Reply:
This module is just temporary and should be loaded along with `ufs_hera.intel` before building the model:
```
module use modulefiles/ 
module load ufs_hera.intel ufs_aerosols_hera.intel
```
All GOCART dependencies will be integrated into hpc-stack and added to the main platform-specific modulefiles after GOCART integration.

Reply:
This file should be deleted